tests/test_tester.py

```diff
@@ -1,14 +1,12 @@
 import time
-import os
 import unittest
+
 import can
 
 try:
-    from queue import Queue
-    from queue import Empty
+    from queue import Empty, Queue
 except ImportError:
-    from Queue import Queue
-    from Queue import Empty
+    from Queue import Empty, Queue
```

Comment:
This was necessary for python 2 and is obsolete now

Reply:
the change itself was also ruff's fault :). fixed :)